### ISSUE TITLE
FreeTypeFont error when trying to run magnum-fontconverter

### ISSUE THREAD

**Issue by neverender** [mosra/magnum#93]
Ok now I'm trying to get fontconverter working.

```
magnum-fontconverter --font FreeTypeFont --converter MagnumFontConverter font.ttf myfont
```
```
Renderer: Intel HD Graphics 3000 OpenGL Engine by Intel Inc.
OpenGL version: 3.3 INTEL-10.0.19
Using optional features:
    GL_ARB_ES2_compatibility
    GL_ARB_separate_shader_objects
    GL_ARB_texture_storage
    GL_ARB_vertex_array_object
    GL_EXT_debug_label
    GL_EXT_debug_marker
    GL_EXT_texture_filter_anisotropic
PluginManager::Manager::load(): cannot open plugin file "/usr/local/lib/magnum/fonts/FreeTypeFont.so": dlopen(/usr/local/lib/magnum/fonts/FreeTypeFont.so, 10): Symbol not found: __ZN6Magnum4Text12AbstractFont10doOpenDataERKNSt3__16vectorINS2_4pairINS2_12basic_stringIcNS2_11char_traitsIcEENS2_9allocatorIcEEEEN7Corrade10Containers14ArrayReferenceIKhEEEENS8_ISG_EEEEf
  Referenced from: /usr/local/lib/magnum/fonts/FreeTypeFont.so
  Expected in: /usr/local/lib/libMagnumText.dylib
 in /usr/local/lib/magnum/fonts/FreeTypeFont.so
```

**Comment by mosra** [comment]
Isn't it possible that you have the plugin compiled with and linked to an older version of Magnum? It complains about the following symbol not found:
```cpp
Text::AbstractFont::openData(const std::vector<std::pair<std::string,
    Containers::ArrayReference<const unsigned char>>>&, Float)
```
but in current master, the function has the following signature:
```cpp
Text::AbstractFont::openData(const std::vector<std::pair<std::string,
    Containers::ArrayReference<const char>>>&, Float)
```

I did this change in fd0bfcb4657ad113efeb44ca36fdc1b2f2af3266 (two months ago), can you try to rebuild the plugins with current Magnum master if it helps?

**Comment by neverender** [comment]
You were right. After updating I'm getting this:

```
Renderer: Intel HD Graphics 3000 OpenGL Engine by Intel Inc.
OpenGL version: 3.3 INTEL-10.0.19
Using optional features:
    GL_ARB_ES2_compatibility
    GL_ARB_separate_shader_objects
    GL_ARB_texture_storage
    GL_ARB_vertex_array_object
    GL_EXT_debug_label
    GL_EXT_debug_marker
    GL_EXT_texture_filter_anisotropic
Populating distance field glyph cache...
Assertion _viewport != Range2Di{} failed in /Users/nick/src/magnum/src/Magnum/AbstractFramebuffer.cpp on line 215
[1]    28343 abort      magnum-fontconverter --font FreeTypeFont --converter MagnumFontConverter  
```

**Comment by mosra** [comment]
Ah, sorry again for another OS X-specific bug :) I assumed that the size of default viewport would never be zero, but apparently that's exactly the case when creating windowless context on OS X. Should be fixed in fbf065645da9b01af61248970610e0ecee32b7dd, please reopen if the problem persists.
